tests/pytest/test_fixtures.py

```diff
@@ -0,0 +1,49 @@
+# coding=utf-8
+#
+# This file is part of Hypothesis, which may be found at
+# https://github.com/HypothesisWorks/hypothesis-python
+#
+# Most of this work is copyright (C) 2013-2017 David R. MacIver
+# (david@drmaciver.com), but it contains contributions by others. See
+# CONTRIBUTING.rst for a full list of people who may hold copyright, and
+# consult the git log if you need to determine who owns an individual
+# contribution.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at http://mozilla.org/MPL/2.0/.
+#
+# END HEADER
+
+from __future__ import division, print_function, absolute_import, \
+    unicode_literals
+
+from hypothesis import example, given
+from hypothesis.strategies import integers
+
+import pytest
+
+
+@pytest.fixture
+def infinity():
+    return float('inf')
+
+
+@given(integers())
+def test_can_mix_fixtures_and_named_strategies(infinity, xs):
```

Comment:
I feel like you have your test names backwards: You passed a positional argument to the one called named and a keyword argument to the one not.